### PR TITLE
Changes to manifest

### PR DESCRIPTION
**Pull request by qinezh**
@vwxyzh @superyyrrzz @ansyral @hellosnow @chenkennt

### CODE REVIEW

**Comment by qinezh** [comment]
old schema:
```json
{ 
  "publish_only_metadata": { 
    "publish_version": "" 
  }, 
  "files": [{ 
    "type": "Conceptual", 
    "original": "openpublishing/docs/partnerdocs/publishedpages.md", 
    "output": { 
      ".mta.json": "partnerdocs/publishedpages.mta.json", 
      ".html": "partnerdocs/publishedpages.html" 
    }, 
    "hashes": { 
      ".mta.json": "DiKaYrFHbcV+rehS+8Qspw==", 
      ".html": "x0MMF9czE05m5jBdHA4y+A==" 
    } 
  }] 
}
```

new schema:
```json
{
  "publish_only_metadata": {
    "publish_version": ""
  },
  "files": [
    {
      "type": "Conceptual",
      "original": "openpublishing/docs/partnerdocs/publishedpages.md",
      "output": {
        ".mta.json": {
          "relative_path": "partnerdocs/publishedpages.mta.json",
          "link_to_path": null,
          "hash": "DiKaYrFHbcV+rehS+8Qspw=="
        },
        ".html": {
          "relative_path": "partnerdocs/publishedpages.html",
          "link_to_path": "W:\\43r6jfy.jrl\\cache\\docfx\\output\\openpublishing\\docs\\partnerdocs\\publishedpages.html",
          "hash": "DiKaYrFHbcV+rehS+8Qspw=="
        }
      }
    }
  ]
}
```

**Comment by qinezh** [comment]
@herohua

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by vwxyzh** [comment]
👍

File: src/Microsoft.DocAsCode.Plugins/ManifestItem.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    public class ManifestItem
+    {
+        [YamlMember(Alias = "type")]
+        [JsonProperty("type")]
+        public string DocumentType { get; set; }
+
+        [YamlMember(Alias = "original")]
+        [JsonProperty("original")]
+        public string OriginalFile { get; set; }
```
Comment (herohua) [inline src/Microsoft.DocAsCode.Plugins/ManifestItem.cs]: use `source_relative_path` instead of `original`

**Comment by herohua** [comment]
:panda_face:

**Pull request merged**